lib/astarte/client/housekeeping/realms.ex

```diff
@@ -47,10 +47,12 @@ defmodule Astarte.Client.Housekeeping.Realms do
     request_path = "realms"
     tesla_client = client.http_client
     query = Keyword.get(opts, :query, [])
+    device_registration_limit = Keyword.get(opts, :device_registration_limit, nil)
```

Comment:
`nil` is already the default, no need to explicitly add it

Reply:
Yep, I removed the default `nil`